{
  "id": 1385,
  "name": "ERROR_CLUSTER_INTERNAL_INVALID_FUNCTION",
  "description": "An internal cluster error occurred. A call to an invalid function was attempted.",
  "external_url": "https://github.oblokos.com/?art=ERROR_CLUSTER_INTERNAL_INVALID_FUNCTION",
  "image": "https://github.oblokos.com/art/ERROR_CLUSTER_INTERNAL_INVALID_FUNCTION.png",
  "opensea_url": "https://opensea.io/assets/matic/0x252f9390feb63aeb00042dd5c4ff84af33f42b85/1385",
  "attributes": [
    {
      "trait_type": "artist",
      "value": "0x5Ae65e30834f61B26A25D9F9F0847c3da11dD391"
    },
    {
      "trait_type": "collection",
      "value": "Metaverse First Contact"
    },
    {
      "trait_type": "error_code",
      "value": "5912 (0x1718)"
    },
    {
      "display_type": "number",
      "trait_type": "frame",
      "value": "14873"
    },
    {
      "display_type": "number",
      "trait_type": "index",
      "value": 0.592
    },
    {
      "display_type": "number",
      "trait_type": "level",
      "value": 0
    }
  ]
}